generator/generator.yml

```diff
@@ -655,3 +655,31 @@ modules:
         ignore: true # Lookup metric
       ifType:
         type: EnumAsInfo
+
+# Draytek Corporation
+#
+# https://www.draytek.com/assets/files/external/DRAYTEK-CORPORATION.mib
+#
+  draytek:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
```

Comment:
Now that we support multiple modules per scrape, we're de-duplicating these to avoid excessive config bloat.

```suggestion
```